kernelci/runtime/__init__.py

```diff
@@ -138,15 +138,16 @@ def get_params(self, job, api_config=None):
         params = {
             'api_config': api_config or {},
             'storage_config': job.storage_config or {},
-            'platform_config': job.platform_config.to_dict() or {},
+            'platform_config': job.platform_config or {},
```

Comment:
What would be the reason not to convert it to dictionary? This already covered `None` case fixed in L149 below

Reply:
The reasons are consistency (other config objects aren't converted that way, and it isn't needed) and code simplicity: the `to_dict()` method isn't part of the `YAMLConfigObject` base class but is defined (for an unknown reason) in [_YAMLObject](https://github.com/kernelci/kernelci-core/blob/main/kernelci/config/base.py#L74).